client/src/main/java/jass/client/controller/LobbyController.java

```diff
@@ -127,7 +162,94 @@ public void initialize(URL location, ResourceBundle resources) {
         findMatch.textProperty().bind(I18nUtil.createStringBinding(findMatch.getText()));
         cancelMatch.textProperty().bind(I18nUtil.createStringBinding(cancelMatch.getText()));
         searching.textProperty().bind(I18nUtil.createStringBinding(searching.getText()));
+    }
+
+    public void startSearchForGame(ActionEvent actionEvent) {
```

Comment:
you're not using this method. Did you test it ?

Reply:
Thanks! Forgot to delete after merging with GUI. Removed.

Reply:
no worries, really good job though!